### DISCUSSION TITLE
[BUG CSS] Impossible to scroll and select images in the product/variant media selection overlay

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#11195] [General] [answered]
Hi guys,
in Saleor  dashboard v3.8.0-dev core v3.8.0-a there's a quite boring bug that doesn't allow to the editor to scroll the media selection box in the case there's many images.

Here the screenshoot where below you can see the wrong overflow and scroll behaviour.
<img width="1078" alt="Screenshot 2022-11-14 at 09 54 18" src="https://user-images.githubusercontent.com/38940340/201617927-6642315b-8548-4e16-9dda-7675eea73647.png">

Is there a way to have a quick patch? 

Thanks and kind regards,

Davide

**Comment by karolkielecki** [comment, marked as answer]
Hi @DidoMarchet, thanks for reporting! We're aware of this bug. We've also created an issue--please follow the status here: https://github.com/saleor/saleor-dashboard/issues/2553

**Reply by DidoMarchet** [reply]
Hi, thanks.
We're waiting for the new release! 
Thanks and kind regards,
Davide

**Reply by DidoMarchet** [reply]
Hi @karolkielecki , I'm  really sorry if I ask but I would like to know if in one of the next release this issue will be solved because it could be restrictive for the data entry.

Thanks and kind regards,

Davide
